### DISCUSSION TITLE
installing Micropython installs tons of unnecessary stuff

### DISCUSSION THREAD

**Discussion by steeley** [Homebrew/discussions#3400] [Getting started]
why is that home-brew installs tons of stuff I don't need?
brew install micropython. (on intel Mac) should be a small file.
brew decides to install a load of full python versions a ton of QT (WTF??) stuff and loads of other useless stuff

has it got corrupted ??

its been installing stuff for the past 15 minutes!! LOL

**Comment by SMillerDev** [comment]
Well, https://formulae.brew.sh/formula/micropython says it depends on python. So it should install that. I expect the other dependencies to be from https://docs.brew.sh/FAQ#why-does-brew-upgrade-formula-or-brew-install-formula-also-upgrade-a-bunch-of-other-stuff

**Reply by steeley** [reply]
thanks I was aware of that, but it just seems to be rather crazy on its enthusiasm for downloading and updating.
I'm going back to doing stuff from source by hand from now on.

**Reply by SMillerDev** [reply]
The longer you don't update your system the more brew has to update when you use it. It's really rather simple.
